test/compiler.test.js

```diff
@@ -1014,6 +1014,176 @@ describe('compiler', function() {
+          appdir.writeFileSync('mixins/PascalCase.js', '');
+          appdir.writeFileSync('mixins/space name.js', '');
+        });
+
+        it('supports classify', function() {
+          var options = { appRootDir: appdir.PATH, normalization: 'classify',
+          mixinDirs: ['mixins'] };
+          var instructions = boot.compile(options);
+
+          var mixins = instructions.mixins;
+          var mixinNames = mixins.map(getNameProperty);
+
+          expect(mixinNames).to.eql(['CamelCase', 'Foo',
+            'PascalCase', 'SpaceName', 'TimeStamps']);
+        });
+
+        it('supports dasherize', function() {
+          var options = { appRootDir: appdir.PATH, normalization: 'dasherize',
+          mixinDirs: ['mixins'] };
+          var instructions = boot.compile(options);
+
+          var mixins = instructions.mixins;
+          var mixinNames = mixins.map(getNameProperty);
+
+          expect(mixinNames).to.eql(['camel-case', 'foo',
+            'pascal-case', 'space-name', 'time-stamps']);
+        });
+
+        it('supports custom function', function() {
+          var normalize = function(name) { return name.toUpperCase(); };
+          var options = { appRootDir: appdir.PATH, normalization: normalize,
+          mixinDirs: ['mixins'] };
+          var instructions = boot.compile(options);
+
+          var mixins = instructions.mixins;
+          var mixinNames = mixins.map(getNameProperty);
+
+          expect(mixinNames).to.eql(['CAMELCASE', 'FOO',
+            'PASCALCASE', 'SPACE NAME', 'TIME-STAMPS']);
+        });
+
+        it('supports none', function() {
+          var options = { appRootDir: appdir.PATH, normalization: 'none',
+          mixinDirs: ['mixins'] };
+          var instructions = boot.compile(options);
+
+          var mixins = instructions.mixins;
+          var mixinNames = mixins.map(getNameProperty);
+
+          expect(mixinNames).to.eql(['camelCase', 'foo',
+            'PascalCase', 'space name', 'time-stamps']);
+        });
+
+        it('supports false', function() {
+          var options = { appRootDir: appdir.PATH, normalization: false,
+          mixinDirs: ['mixins'] };
+          var instructions = boot.compile(options);
+
+          var mixins = instructions.mixins;
+          var mixinNames = mixins.map(getNameProperty);
+
+          expect(mixinNames).to.eql(['camelCase', 'foo',
+            'PascalCase', 'space name', 'time-stamps']);
+        });
+
+        it('defaults to classify', function() {
+          var options = { appRootDir: appdir.PATH,
+            mixinDirs: ['mixins'] };
+          var instructions = boot.compile(options);
+
+          var mixins = instructions.mixins;
+          var mixinNames = mixins.map(getNameProperty);
+
+          expect(mixinNames).to.eql(['CamelCase', 'Foo',
+            'PascalCase', 'SpaceName', 'TimeStamps']);
+        });
+      });
+
+      it('throws error for invalid normalization format', function() {
```

Comment:
I think this should be nested under "name normalization", for example as "(it) throws error for invalid format"